pkg/strategy/atrpin/strategy.go

```diff
@@ -199,21 +200,3 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 
 	return nil
 }
-
-func logErr(err error, msgAndArgs ...interface{}) bool {
```

Comment:
i think we need this func, maybe move this to util?

Reply:
OK, I will drop the commit and create another PR to move `logErr`.